app/views/users/_user.html.erb

```diff
@@ -0,0 +1,137 @@
+<div class="media">
+  <%= image_tag user.avatar_image, class: "mr-3 rounded-circle", width: 128 %>
+
+  <div class="media-body">
+    <h1 class="mt-0 mb-3 display-4">
+      <%= user.username %>
+    </h1>
+
+    <div class="row mb-4">
+      <div class="col">
+        <%=
+          link_to "#",
+            data: { 
+              toggle: "modal", 
+              target: "##{dom_id(user)}_followers"
+            } do %>
+
+          <span class="font-weight-bold">
+            <%= user.followers.size %>
+          </span>
+          
+          followers
+
+        <% end %>
+      </div>
+
+      <div class="col">
+        <%=
+          link_to "#",
+            data: { 
+              toggle: "modal", 
+              target: "##{dom_id(user)}_following"
+            } do %>
+
+          <span class="font-weight-bold">
+            <%= user.leaders.size %>
+          </span>
+          
+          following
+
+        <% end %>
+      </div>
+
+      <div class="col">
+        <%=
+          link_to "#",
+            data: { 
+              toggle: "modal", 
+              target: "##{dom_id(user)}_pending"
+            } do %>
+
+          <span class="font-weight-bold">
+            <%= user.pending.size %>
+          </span>
+          
+          pending
+
+        <% end %>
+      </div>
+    </div>
+
+    <div class="row mb-3">
+      <div class="col">
+        <% follow_request = current_user.received_follow_requests.find_by(sender: @user) %>
+        <% if follow_request %>
+          <% if follow_request.pending? %>
+            <% follow_request.status = "accepted" %>
+
+            <%= form_with(model: follow_request) do |form| %>
+              <%= form.hidden_field :status %>
+
+              <%= form.submit "Accept follow request", class: "btn btn-outline-success btn-block mb-3" %>
+            <% end %>
+
+            <%= link_to follow_request, method: :delete, class: "btn btn-outline-danger btn-block" do %>
+              Reject follow request
+            <% end %>
```

Comment:
Since there is a `"rejected"` status, instead of deleting the follow request, should this be another update?

Reply:
Ah yes, that would be better. Nice catch